website/index.html

```diff
@@ -0,0 +1,97 @@
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta http-equiv="Content-Style-Type" content="text/css">
+  <base target="_blank">
+  <meta name="description" content="Official website for the IoT project Wearable My Foot" />
+  <title>Wearable My Foot</title> 
+  <meta property="og:title" content="Wearable My Foot"/>
+  <meta property="og:site_name" content="wearablemyfoot.tech"/>
+  <meta property="og:description" content="Wearable My Foot is an OpenSource IoT project that combines DIY hardware and machine learning to create a 'state of the art' wearable device for your feet. Powered by Manchester based nerds Fuzzy Labs: fuzzylabs.ai" />
+  <meta name="viewport" content="width=device-width">
+
+  
+  <style type="text/css">
+  
+    a:link {color:#CE2029; text-decoration:none;}
+    a:visited {color:#CE2029; text-decoration:none;}
+    a:hover {text-decoration:underline;}
+    
+    pre {
+      font: 11.0px 'SourceCodePro Italic';
+      width: 585px;
+      position: relative;
+      white-space: -moz-pre-wrap; /* Mozilla, supported since 1999 */
+      white-space: -pre-wrap; /* Opera */
+      white-space: -o-pre-wrap; /* Opera */
+      white-space: pre-wrap; /* CSS3 - Text module (Candidate Recommendation) http://www.w3.org/TR/css3-text/#white-space */
+      word-wrap: break-word; /* IE 5.5+ */
+    }
+    
+  @font-face {
+    font-family: 'SourceCodePro Italic';
+    src:    url('SourceCodePro-Italic.ttf') format('truetype');
+    font-weight: 400;
+    font-style: normal;
+    }
+         
+  </style>
+ 
+
+</head>
+
+<body>
+
```

Comment:
Tidy up whitepsace in this file. No more than 2 blank lines in a row (except for the content of course)